applications/NXem.nxdl.xml

```diff
@@ -0,0 +1,1693 @@
+                    is available in the literature for example:
+                    
+                    * `S. Richter et al. &lt;https://doi.org/10.1088/1757-899X/109/1/012014&gt;`_
+                    * `J. Bünger et al. &lt;https://doi.org/10.1017/S1431927622000083&gt;`_
+                    * `J. F. Ziegler et al. &lt;https://doi.org/10.1007/978-3-642-68779-2_5&gt;`_
+                    
+                    Instances should use interaction_volume as a name prefix.
+                </doc>
+                <group type="NXdata" recommended="true"/>
+                <group type="NXprocess" recommended="true"/>
+            </group>
+        </group>
+        <!--relevant research result post-processed for specific community methods
+but normalized in its representation ready to be consumed for
+research data management systems-->
+        <group name="ROI" type="NXobject" nameType="any" minOccurs="0" maxOccurs="unbounded">
+            <doc>
+                A region-of-interest analyzed either during or after the session for which specific
+                processed data are available. Instances should use roi as a name prefix.
+                
+                This concept is related to term `Region Of Interest`_ of the EMglossary standard.
+                
+                .. _Region Of Interest: https://purls.helmholtz-metadaten.de/emg/EMG_00000042
+            </doc>
+            <!--as soon as one entry is here constrained further
+an RDM can be sure to find specific pieces of information in a
+specific way but then every user of this application definition
+is required to provide such information in this way!-->
+            <group name="img" type="NXem_img" optional="true">
+                <group type="NXimage" minOccurs="1" maxOccurs="unbounded">
+                    <doc>
+                        Instances should use image as a name prefix.
+                        Each NXimage instance must use only one image or stack instance.
+                    </doc>
+                    <field name="imaging_mode" type="NX_CHAR"/>
+                </group>
+            </group>
+            <!--connection point to store serialized representations of microstructures
+microstructureID(NXmicrostructure):
+exists: optional-->
+            <group name="ebsd" type="NXem_ebsd" optional="true">
+                <group name="gnomonic_reference_frame" type="NXcoordinate_system" optional="true">
+                    <field name="alias" type="NX_CHAR" optional="true"/>
+                    <field name="type" type="NX_CHAR"/>
+                    <field name="handedness" type="NX_CHAR"/>
+                    <field name="origin" type="NX_CHAR"/>
+                    <field name="x_direction" type="NX_CHAR"/>
+                    <field name="y_direction" type="NX_CHAR"/>
+                    <field name="z_direction" type="NX_CHAR"/>
+                </group>
+                <group name="pattern_centre" type="NXprocess" recommended="true">
```

Comment:
Ooph, center vs. centre rears its ugly head yet again...

Reply:
Yeah I should use American spelling

Reply:
Has been addressed in the new draft on the fairmat side, ones I am done with all changes, will update this PR here as well